### PR TITLE
Add custom properties for swish and autogiro

### LINKED ISSUE #933: Add custom properties to plausible for sweden

### PR DESCRIPTION
**Pull request by EliseAndresen**
_Issue description here_

- closes #933 

---

Tested on devices

- [ ] Desktop 💻
- [ ] Mobile 📱

Tests

- [ ] All tests are running ✔️
- [ ] Test are updated 🧪
- [ ] Code Review 👩‍💻
- [ ] QA 👌

Checkpoints

_Check these to flag for a more thurough review, as they could be potentially breaking changes_

- [ ] Packages updated
- [ ] Other infrastructure updated (such as node version or similar)

⏲️ Time spent on CR:

⏲️ Time spent on QA:

### CODE REVIEW

File: components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export const RecurringBankDonationForm: React.FC<{
   const plausible = usePlausible();
 
   const onSubmit = (e: React.MouseEvent<Element, MouseEvent>) => {
-    plausible("DraftAvtalegiro");
+    plausible("DraftAutogiro");
```
Comment (fellmirr) [inline components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx:17]: Should probably be in the AutoGiro form

**Review by fellmirr — CHANGES_REQUESTED** [review]
Looks pretty straight forward! Just the one small thing with wrong form.

File: components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx (hunk 1 above)
Comment (EliseAndresen) [reply components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx]: This has been reset :D

File: components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const RecurringBankDonationForm: React.FC<{
   const plausible = usePlausible();
 
   const onSubmit = (e: React.MouseEvent<Element, MouseEvent>) => {
-    plausible("DraftAvtalegiro");
+    plausible("DraftAvtaleGiro");
```
Comment (fellmirr) [inline components/shared/components/Widget/components/panes/PaymentPane/AvtaleGiro/RecurringForm.tsx:17]: Denne burde ikke endres, vi kan bare la den være som den er med liten g

**Review by fellmirr — CHANGES_REQUESTED** [review]
Very small fix to required, otherwise seems to work!

**Pull request merged**